### PR TITLE
feat: payment methods selector add currency notice

### PR DESCRIPTION
**Pull request by frosso**
Fixes #

#### Changes proposed in this Pull Request

Adding the notice about enabled currencies in the payment methods selector modal.

#### Testing instructions

- Ensure you have "Enable UPE settings changes" enabled
- Ensure you have "Enable UPE checkout" enabled
- Ensure you have "Enable Customer multi-currency" enabled
- Ensure your store's currency is set to "USD" and you don't have multiple currencies enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Click the "Add payment method" button
- Select one of the available payment methods
- If an EU payment method is selected, the notice will be displayed

If the "multi currency" flag is disabled, you'll get a notice.
But we'll launch at the same time, so no biggie.
Screen Shot 2021-07-16 at 1 42 01 PM

-------------------

- [ ] Added changelog entry (or does not apply)
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

**Comment by harriswong** [comment]
👀

File: client/settings/payment-methods-selector/index.js [hunk 1]
```diff
@@ -10,12 +10,14 @@ import { useState, useCallback } from '@wordpress/element';
  * Internal dependencies
  */
 import {
+	useCurrencies,
 	useEnabledPaymentMethodIds,
 	useGetAvailablePaymentMethodIds,
-} from 'data';
+} from '../../data';
```
Comment (harriswong) [inline client/settings/payment-methods-selector/index.js:16]: The original `from 'data'` should still work right? It should resolves to `client/`. It makes no difference, but curious on the change. Also thinking if it's more consistent with the other places by using just `from 'data'`.

https://github.com/Automattic/woocommerce-payments/blob/4b8fd1bd27acc1232a0e3d4abd2078f11c19f586/webpack.config.js#L83

Comment (harriswong) [reply client/settings/payment-methods-selector/index.js:16]: I guess it applies also to all the `../../components` below too.

**Review by harriswong — APPROVED** [review]
Tested the following:
- with existing EUR, notice doesn't show. 
- with just USD, notice shows.

👍

File: client/settings/payment-methods-selector/index.js (hunk 1 above)
Comment (frosso) [reply client/settings/payment-methods-selector/index.js:16]: Yeah it does, it's just a matter of preference :) 
```suggestion
} from 'data';
```

**Pull request merged**